content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md

```diff
@@ -275,3 +279,15 @@ Such a controller is not a secure mechanism unless it not only verifies the Comm
 in the CSR but also verifies the requested IPs and domain names. This would prevent
 a malicious actor that has access to a kubelet client certificate to create
 CSRs requesting serving certificates for any IP or domain name.
+
+## Troubleshooting kubelet client certificates rotation {#kubelet-client-cert}
+
+kubelet handles its own client certificates, located in `/var/lib/kubelet/pki/`, unless [rotation](/docs/tasks/tls/certificate-rotation/) is manually disabled. For this reason, `kubeadm certs` subcomands ignore kubelet certificates. The kubelet auto-renew process has [been](https://github.com/kubernetes/kubeadm/issues/581) [reported](https://github.com/kubernetes/kubernetes/issues/102556) to fail, which can break the control-plane. One sign of this is `x509: certificate has expired or is not yet valid` errors in apiserver logs. To fix the issue:
+1. Check that `/var/lib/kubelet/pki/kubelet-client-current.pem` is a symlink. If not, backup and delete it. kubelet will be recreate it.
+2. Backup and delete `/etc/kubernetes/kubelet.conf`, regenerate with `kubeadm init phase kubeconfig kubelet`, pass the `--config` file you used with `kubeadm init`, or if the cluster is too old, create a config file, or don't use one and manually edit the new `kubelet.conf` if necessary.
+3. Restart the `kubelet` service and delete the `kube-apiserver` POD. Wait for `kubelet-client-current.pem` to be created/updated.
+4. Run `kubeadm init phase kubelet-finalize all` using `--config` file if necessary.
+5. Restart kubelet again
+6. Sometimes it may be necessary to `kubeadm certs renew all`, and then restart kubelet and kube-system PODs (or reboot the node).
```

Comment:
i don't think this step is required for troubleshooting the kubelet client cert on a particular node.
it would update all CP certificates for this node, which means on HA setups the user must do the same on all nodes.